ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -315,21 +316,6 @@ aaa_server_groups:
       - server: < host1_ip_address >
 ```
 
-#### CVX
```

Comment:
So I am not sure why this is gone?

Reply:
This was from a rebase, let me dig into this

Reply:
This must've been a weird error on my end. I'll get this added back in.

Reply:
Added this back in, apologies

Reply:
thank you!